src/views/Sales/SalesAdd.vue

```diff
@@ -169,35 +149,60 @@ const isFormValid = async () => {
 };
 
 const isProductsValidated = () => {
-  if (productsForSale.value.length <= 0) {
-    snackbarProvider.showErrorSnackbar("Please select a product!");
+  const isProductsValid = sellObject.products.value;
+  const isResourcesValid = sellObject.resources.value;
```

Comment:
Consider renaming as suggested below. These names are lying because they are not a result of any check. They are the products and resources that are going to be sold if I get it correctly
```suggestion
  const selectedProducts = sellObject.products.value;
  const selectedResources = sellObject.resources.value;
```

Reply:
updated